src/App.tsx

```diff
@@ -1,19 +1,20 @@
-import reactLogo from '@/assets/react.svg';
-
+import { AboutMeComponent } from './components/about/AboutMe.component.tsx';
+import { FooterComponent } from './components/footer/Footer.component.tsx';
 import { HeaderComponent } from './components/header/Header.component.tsx';
 
 import './App.css';
 
 function App() {
+    const name: string = `Roman Orlovskyi`;
```

Comment:
If this are cont, then use `UPPERCASE`
```suggestion
    const USER_NAME: string = `Roman Orlovskyi`;
```